### DISCUSSION TITLE
Pivpn server slows down traffice by a factor of 3

### DISCUSSION THREAD

**Discussion by MichaelGofron** [pivpn/pivpn#1657] [Support]
Sorry for not using the template.

 I've set up a pivpn server from the east coast. I'm testing from the west coast in oregon and california and seeing a dropoff in speed when connecting to the pivpn server. 

The pi server is physically connected by fiber ethernet with 540 mbs down, 920 mbs up.
At the place I tested in california it showed 500 mbs down, 10 mbs up. With vpn it went to 130 mbs down, 4 mbs up. Any ideas why such a dropoff? Is it just because of the distance? I'd been under the impression there shouldn't be any major drop off if the speeds on both sides of the connection are so stable.

**Comment by thewriteway** [comment]
are you using OpenVPN or Wireguard mode? WG mode would be close to physical speeds where OpenVPN is much slower.

**Reply by MichaelGofron** [reply]
I am using Wireguard not OpenVPN

**Comment by coolapso** [comment]
> Sorry for not using the template. 

Its fine for this case .. moving it to discussion tho... 

There are various factors that can affect this, have you tried searching for similar closed issues? there were a few of those over the years.
